docs/05_integrations/django.md

```diff
@@ -114,3 +114,114 @@ class MyGraphQLView(GraphQLView):
 
 In this case we are doing the default processing of the result, but it can
 be tweaked based on your needs.
+
+# Async Django
+
+Strawberry also provides an async view that you can use with Django 3.1+
+
+```python
+from django.urls import path
+
+from strawberry.django.views import AsyncGraphQLView
+
+from api.schema import schema
+
+urlpatterns = [
+    path("graphql/", AsyncGraphQLView.as_view(schema=schema)),
+]
+```
+
+You'd also need to add `strawberry.django` to the `INSTALLED_APPS` of your
+project, this is needed to provide the template for the GraphiQL interface.
```

Comment:
BTW this isn't true because the template is loaded in using the Template loader directly: https://github.com/strawberry-graphql/strawberry/blob/7f2e848717050abc26d6b664459757761cb8f078/strawberry/django/views.py#L69

Reply:
really? :O Will check that!

Reply:
Actually it's these lines that do it:
https://github.com/strawberry-graphql/strawberry/blob/7f2e848717050abc26d6b664459757761cb8f078/strawberry/django/views.py#L71-L79